app/experimenter/nimbus-ui/src/components/AppLayoutExperimentEdit/index.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Nav from "react-bootstrap/Nav";
+
+import "./index.scss";
+
+type AppLayoutExperimentEditProps = {
+  children: React.ReactNode;
+};
+
+export const AppLayoutExperimentEdit = ({
+  children,
+}: AppLayoutExperimentEditProps) => {
+  return (
+    <Container
```

Comment:
tl;dr: Am I going in the right direction here? Think I'm confusing myself here and making this into a bigger issue. Might also just be a migraine talking.

How pixel-perfect to the Figma layout do we want to be? Might be that I'm rusty on responsive front-end UI work, but also this is my first encounter with React Bootstrap. How much custom CSS do we want vs just going with the Bootstrap flow?

Reply:
We sort of talked about this earlier but yeah, this is LGTM! It's not rendering as `<nav>` with `<ul>`/`<li>`s, but that was actually going to be a separate issue anyway and you've just started it up for us.

Is there a way to add a sort of `max-width` on the sidebar? It keeps expanding outward with a larger screen.

Reply:
Looks like I can add a `xl="2"` for the largest breakpoint. That might do the trick